cypress/integration/message-renderer/adaptiveCard.spec.ts

```diff
@@ -0,0 +1,135 @@
+                                "text": "Matt Hidinger",
+                                "weight": "bolder",
+                                "wrap": true
+                              },
+                              {
+                                "type": "TextBlock",
+                                "spacing": "none",
+                                "text": "Created {{DATE(2017-02-14T06:08:39Z, SHORT)}}",
+                                "isSubtle": true,
+                                "wrap": true
+                              }
+                            ]
+                          }
+                        ]
+                      }
+                    ]
+                  },
+                  {
+                    "type": "Container",
+                    "items": [
+                      {
+                        "type": "TextBlock",
+                        "text": "Now that we have defined the main rules and features of the format, we need to produce a schema and publish it to GitHub. The schema will be the starting point of our reference documentation.",
+                        "wrap": true
+                      },
+                      {
+                        "type": "FactSet",
+                        "facts": [
+                          {
+                            "title": "Board:",
+                            "value": "Adaptive Card"
+                          },
+                          {
+                            "title": "List:",
+                            "value": "Backlog"
+                          },
+                          {
+                            "title": "Assigned to:",
+                            "value": "Matt Hidinger"
+                          },
+                          {
+                            "title": "Due date:",
+                            "value": "Not set"
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                ],
+                "actions": [
+                  {
+                    "type": "Action.ShowCard",
+                    "title": "Comment",
+                    "card": {
+                      "type": "AdaptiveCard",
+                      "body": [
+                        {
+                          "type": "Input.Text",
+                          "id": "comment",
+                          "isMultiline": true,
+                          "placeholder": "Enter your comment"
+                        }
+                      ],
+                      "actions": [
+                        {
+                          "type": "Action.Submit",
+                          "title": "OK"
+                        }
+                      ]
+                    }
+                  }
+                ]
+              }              
+        }
+    }
+      
+
+    it('uses an alternate theme for messages when providing colorScheme setting', () => {
```

Comment:
you're testing that the adaptivecard gets rendered, but the test name is not matching that

Reply:
Thanks, forgot to change that before.